app/src/main/java/org/fossasia/openevent/general/event/EventDetailsViewModel.kt

```diff
@@ -27,6 +32,22 @@ class EventDetailsViewModel(private val eventService: EventService, private val
     private val mutableEvent = MutableLiveData<Event>()
     val event: LiveData<Event> = mutableEvent
 
+    fun fetchEventSpeakers(id: Long) {
+        speakerService.fetchSpeakersForEvent(id)
+            .subscribeOn(Schedulers.io())
+            .observeOn(AndroidSchedulers.mainThread())
+            .subscribe ({
+            }, {
```

Comment:
Add a comment in empty block saying `// Do Nothing`

Reply:
updated